run-tests.sh

```diff
@@ -21,4 +21,4 @@
 # submit itself to any jurisdiction.
 
 sphinx-build -qnN docs docs/_build/html
-prettier reana-ui/src/**/*.js --list-different
+#prettier reana-ui/src/**/*.js --list-different
```

Comment:
To be removed on https://github.com/reanahub/reana-ui/pull/11.

Reply:
That's what I mentioned in the commit log :smile: